external/fv3fit/fv3fit/sklearn/_random_forest.py

```diff
@@ -86,15 +100,20 @@ def load(cls, path: str) -> "SklearnWrapper":
 
 
 class _RegressorEnsemble:
-    """Ensemble of regressors that are incrementally trained in batches
-
+    """
+    Ensemble of RandomForestRegressor objects that are each trained on a separate
+    batch of data.
     """
 
     def __init__(
-        self, base_regressor, regressors: Sequence[sklearn.base.BaseEstimator] = None,
+        self,
+        base_regressor,
+        n_jobs,
```

Comment:
How does this new parameter interact with the serialization? Does a trained model have this value baked in, in such a way that it cannot be trained/applied with a different number of jobs than it was trained with?

Reply:
The parameter is only used in the training. The value we actually pass to the RandomForestRegressor is None, which says "use the joblib.parallel context you're in, or one core if you're not in one". I assume that means if you predicted inside a joblib.parallel context it would use it, but I don't know what the API is.

The parameter gets saved and loaded during serialization, but only for the reason base_regressor is - because this is an Estimator and Predictor. Both base_regresor and n_jobs should not be used when this class is used only as a Predictor.